.github/workflows/publish.yml

```diff
@@ -0,0 +1,33 @@
+on:
+  push:
+    branches:
+      - main
+      - ci
+  schedule:
+    - cron: 0 0 * * *
+name: Build & Publish
```

Comment:
style: Please can we put the `name` at the top and have an empty line between each top-level entry in the YAML file.